README.md

```diff
@@ -6,6 +6,9 @@ range of moving assets (such as vehicles) and visualize them on a live map. The
 applications use a mixture of technologies - Android, Firebase,
 Google Maps, GTFS (General Transit Feed Specification), and more.
 
+**Note:** This application implements asset tracking. For use in a production environment, you therefore need a [Google Maps APIs Premium Plan license](
+https://developers.google.com/maps/pricing-and-plans/). For more information, see the [Google Maps APIs terms of service](https://developers.google.com/maps/terms#section_10_4).
```

Comment:
Consider dropping the comma: 

For more information see the Google Maps APIs terms of service.